docs/release.rst

```diff
@@ -98,9 +98,9 @@ Bug Fix Release
 This describes how to create a new bug fix release, using incrementing from 0.8.0 to 0.8.1 as an example. This assumes that there is already a branch for a minor release called ``0.8``.
 
 #. To create a bug fix release (version 0.8.1), we need to backport all the bug fix commits on the ``master`` branch that have been added since the last bug fix release onto the ``0.8`` branch. For each bug fix PR on ``master``, we need to create a PR against the ``0.8`` branch based on a branch of ``0.8`` that has cherry-picked the commits from the original PR. The title of the PR should start with [BACKPORT].
-#. Make and merge a PR against ``0.8`` (but not ``master``) that increments the version in each ``setup.py`` file to ``0.8.1``. Then wait for the ``0.8`` branch to be built by Travis and the ``0.8`` Docker images to be published to Quay. If that is successful, we can proceed to the next steps of actually publishing a release.
+#. Make and merge a PR against ``0.8`` (but not ``master``) that increments the version in each ``setup.py`` file to ``0.8.1``. Then wait for the ``0.8`` branch to be built by GitHub Actions and the ``0.8`` Docker images to be published to Quay. If that is successful, we can proceed to the next steps of actually publishing a release.
 #. Using the Github UI, make a new release. Use ``0.8.1`` as the tag, and the ``0.8`` branch as the target.
-#. The image for ``0.8`` is created automatically by Travis, but we need to manually create images for ``0.8.1``. For this you will need an account on Quay  under the Azavea organization.
+#. The image for ``0.8`` is created automatically by GitHub Actions, but we need to manually create images for ``0.8.1``. For this you will need an account on Quay  under the Azavea organization.
```

Comment:
Is this still true? It looks like we should automatically build and push minor version images just fine.

Reply:
Cool, I'll remove that bit from the documentation. Thanks!

Reply:
After reading this more closely, it is still technically true. With how we currently tag releases, we will automatically release images for `0.8` based on the branch, and `v0.8.1` based on our tags prefixed with `v`, but since there's no `0.8.1` branch or tag, that will still need to be done manually.

Is having both `0.8.1` and `v0.8.1` actually valuable though? I do see both tags present on Quay.io, so we've been doing it, but I don't know if there was intent behind that.

Reply:
I don't think that having both 0.8.1 and v0.8.1 is valuable. If we are automatically releasing the bug fix versions based on tags that's good enough.